geode-core/src/main/java/org/apache/geode/cache/client/internal/QueryOp.java

```diff
@@ -66,7 +66,7 @@ private QueryOp() {
   /**
    * Note: this class is extended by CreateCQWithIROpImpl.
    */
-  protected static class QueryOpImpl extends AbstractOp {
+  public static class QueryOpImpl extends AbstractOp {
```

Comment:
Why does this need to be public now?  I suppose it is part of an internal package, so does that mean we don't need to worry about this being a public API change?  Mainly I'm just curious why this became public, I'm not seeing  the corresponding usage that requires it to be public in this PR.

Reply:
There is a class in the `geode-cq` that inherits from this inner class. Tt was able to before because it was in the same package but to support modules we can't split packages across jars anymore. In order for the `geode-cq` class to access this class it must now be public. Since it is "internal" it should not be considered public API. In fact it isn't exported in a way that any module other than `geode-cq` can access it in the `module-info.java`.